superset/viz.py

```diff
@@ -396,8 +396,9 @@ def cache_key(self, query_obj: QueryObjectDict, **extra: Any) -> str:
         cache_dict = copy.copy(query_obj)
         cache_dict.update(extra)
 
-        for k in ["from_dttm", "to_dttm"]:
-            del cache_dict[k]
+        for k in ["from_dttm", "to_dttm", "inner_from_dttm", "inner_to_dttm"]:
```

Comment:
@etr2460 can you also make this change in `viz_sip38.py`? Also we should probably update the comment on line 392 to reflect the change.

Reply:
According to @villebro he'd pull any viz changes into viz_sip38 sometime in the future. is that still the case? will update the comment

Reply:
Yes, don't worry about it, I'll be making updates to some of the SIP-38 affected files soon and can take care of these changes during that update.